packages/elements/scripts/cmd/test.mjs

```diff
@@ -1,94 +1,36 @@
 #!/usr/bin/env node
 import { execSync } from 'node:child_process';
-import { DefaultBrowsers, BrowserStack } from '../../browsers.config.mjs';
-import {
-  getElements,
-  info,
-  errorHandler,
-  PACKAGE_NAME
-} from '../helpers/index.mjs';
+import { useTestOptions } from '../../../../scripts/tests/cli-options.mjs';
+import { getElements, errorHandler } from '../helpers/index.mjs';
+import { hideBin } from 'yargs/helpers';
+
+const elements =  ['elements', ...getElements()];
 
-const elements = ['all', 'utils', ...getElements()];
 export const command = 'test [element]';
-export const desc = 'Test package';
+export const desc = 'Test elements';
 export const builder = yargs => {
   yargs
     .positional('element', {
       desc: 'Element name',
       type: 'string',
       choices: elements
     })
-    .option('watch', {
-      alias: 'w',
-      type: 'boolean',
-      default: false,
-      description: 'Run test and watch file change'
-    })
-    .option('update-snapshots', {
-      alias: 's',
-      type: 'boolean',
-      default: false,
-      description: 'Update and prune snapshots'
-    })
-    .option('browsers', {
-      alias: 'b',
-      type: 'array',
-      choices: DefaultBrowsers,
-      description: 'Specific browser(s) to run units test'
-    })
-    .option('browserstack', {
-      type: 'array',
-      alias: 'bs',
-      choices: BrowserStack.availableBrowsers,
-      description: 'Run units test on BrowserStack and specific browser(s)'
-    })
-    .requiresArg('browserstack')
-    .option('include-coverage', {
-      type: 'boolean',
-      default: true,
-      description: 'Include coverage testing'
-    })
-    .option('output', {
-      type: 'string',
-      alias: 'o',
-      default: 'full',
-      choices: ['full', 'minimal'],
-      description: 'Print output to the console'
-    })
     .completion('completion', () => elements);
+
+  // Use shared test options for the CLI
+  useTestOptions(yargs);
 };
 export const handler = (argv) => {
-
-  const element = argv.element || 'all';
-  const watch = argv.watch;
-  const snapshots = argv.updateSnapshots;
-  const browsers = argv.browsers ? argv.browsers.join(' '): '';
-  const browserstack = argv.browserstack ? argv.browserstack.join(' ') : '';
-
-  info(watch ? `Start Dev Server: ${ element }` : `Test: ${ element }`);
-
-  if (snapshots) {
-    info(`Update and prune snapshots: ${ element }`);
-  }
+  // Remove command and forward params only to main test file
+  let params = hideBin(process.argv).slice(1)
```

Comment:
yargs provides arguments for us with `argv` param. We should use it instead.

Reply:
The argv form yargs is already converted to an object that makes it harder to forward to the child process